apps/studio.giselles.ai/.env.example

```
@@ -1,3 +1,42 @@
+# ==============================================================================
+
+# PostgreSQL connection string
+# For local development: postgres://user:password@localhost:5432/giselle
+# @see https://vercel.com/docs/storage/vercel-postgres#getting-started
+POSTGRES_URL=
+
+# GitHub App credentials (all five are validated at boot)
+# @see https://docs.github.com/en/apps/creating-github-apps
+GITHUB_APP_ID=
+GITHUB_APP_PRIVATE_KEY=
+GITHUB_APP_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_APP_CLIENT_ID key should go before the GITHUB_APP_ID key

(UnorderedKey)